packages/teleport/src/Users/useUsers.ts

```diff
@@ -78,7 +78,7 @@ export default function useUsers() {
 
   useEffect(() => {
     function fetchRoles() {
-      if (ctx.isRolesEnabled()) {
+      if (ctx.getFeatureAccess().roles) {
```

Comment:
These are feature flags, not the access. Feature flags are used to disable/enable parts of functionality. It could be because of permissions or because of other reasons. For example, workflows are not available to OSS users not because of access but because of the product type.

Reply:
got it, i see the difference now